data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2124,12 +2124,12 @@
 
         [message]
             speaker=Abhai
-            message= _ "(<i>pales if possible</i>) Malifor...! Finally someone here in force to deal with that menace!"
+            message= _ "Malifor...! Finally someone here in force to deal with that menace!"
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "Can you tell us where he is? And, if I may ask, who are — or rather, were — you?"
+            message= _ "(<i>To himself</i>) <i>Pronouncing Malifor’s name seems to have made the apparition get even more pale, if that is at all possible.</i> (<i>Looking at the wraith</i>) Can you tell us where he is? And, if I may ask, who are — or rather, were — you?"
```

Comment:
Still a bit wordy, maybe something like this is better: 

```suggestion
            message= _ "<i>Is it just me or did this spirit get even paler?</i>  Can you tell us where he is? And, if I may ask, who are — or rather, were — you?"
```

Reply:
Ah, I understand, in that case I would reduce it even further, to something like:

`Did this spirit get even paler?`